src/main/java/core/basesyntax/utils/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.utils;
+
+import core.basesyntax.geometry.Color;
+import core.basesyntax.geometry.Figure;
+import core.basesyntax.geometry.figures.Circle;
+import core.basesyntax.geometry.figures.IsoscelesTrapezoid;
+import core.basesyntax.geometry.figures.Rectangle;
+import core.basesyntax.geometry.figures.RightTriangle;
+import core.basesyntax.geometry.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final String DEFAULT_CIRCLE_COLOR = Color.WHITE.name();
+    private static final int MAX_NUMBER_FOR_SIDES = 10;
+    private static final int MIN_NUMBER_FOR_SIDES = 1;
+    private static final int FIGURES_NUMBER = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private int side = random.nextInt(MAX_NUMBER_FOR_SIDES + MIN_NUMBER_FOR_SIDES);
```

Comment:
use it locally where you need it, you can create a method that will be returning a random number